zerver/views/development/integrations.py

```diff
@@ -0,0 +1,54 @@
+import os
+import ujson
+from typing import Dict
+
+from django.http import HttpRequest, HttpResponse
+from django.shortcuts import render
+from django.utils.translation import ugettext as _
+
+from zerver.lib.integrations import WEBHOOK_INTEGRATIONS
+from zerver.lib.request import has_request_variables, REQ
+from zerver.lib.response import json_success, json_error
+
+
+ZULIP_PATH = os.path.join(os.path.dirname(os.path.abspath(__file__)), '../../../')
+
+
+def get_webhook_integrations():
+    return [integration.name for integration in WEBHOOK_INTEGRATIONS]
```

Comment:
Shouldn't this function have mypy annotations? The build should have complained about that.

Reply:
I'll be polishing up everything like this as this leaves the WIP state :)